applications/GettingStartedWith/HtmlEditor/index.js

```diff
@@ -0,0 +1,78 @@
+$(function () {
+    $("#html-editor").dxHtmlEditor({
+        valueType: "html",
+        toolbar: {
+            items: [
+                "undo",
+                "redo",
+                "separator",
+                {
+                    name: "size",
+                    acceptedValues: [
```

Comment:
```suggestion
                    formatName: "size",
                    formatValues: [
```
Before: 
![image](https://user-images.githubusercontent.com/33040934/117483834-eea53580-af6e-11eb-89f3-7181cad7eb63.png)
After:
![image](https://user-images.githubusercontent.com/33040934/117483848-f4028000-af6e-11eb-8680-ab0058f411d4.png)

Reply:
`formatName` and `formatValues` were renamed to `name` and `acceptedValues` in v21.1. The items aren't displayed because DevExtreme sources in HG are outdated.